server_ftp/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+This module enables connections via FTP, SFTP, FtpLs (wip).
+Operations such as retrieving a file, checking if a file exists, are all centralized
+in an Abstract python class, so the user can operate without even noticing what protocol
+they are using. For more exquisite operations, the user may retrieve a pure connection object in a straighforward way.
+
+TODOS:
+* Implement FtpLS
```

Comment:
This should be trivial. Just base the FTPTLSServer class on FTPServer class, but override the connect method.

Reply:
Yeah, basically it is already there (well not here there, but somewhere in my forked repo). Just haven't tested it yet